src/pages/Home/left-menu.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from "react";
```

Comment:
Let's move those file to container, create a LeftMenu Container and also rename this to LeftMenu.js